int32 robot_command
int32 object_menu
